core/domain/html_cleaner.py

```diff
@@ -64,8 +78,14 @@ def filter_a(name, value):
 def clean(user_submitted_html):
     """Cleans a piece of user submitted HTML.
 
-    This only allows HTML from a restricted set of tags, attrs and styles. It
-    strips out unrecognized tags.
+    This only allows HTML from a restricted set of tags, attrs and styles.
+    
+    Args:
+        user_submitted_html: A HTML file submitted by user.
+    
+    Returns:
+        The HTML file submitted by user after cleaning out unrecognized
```

Comment:
str. The HTML string submitted by the user, after stripping out unrecognized tags and attributes.